CondTools/SiPixel/test/SiPixelVCalReader_cfg.py

```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env cmsRun
+# Author: Izaak Neutelings (March 2020)
+#from __future__ import print_function
+import FWCore.ParameterSet.Config as cms
+from Configuration.StandardSequences.Eras import eras
+
+# SETTINGS
+run       = 313000
+era       = eras.Run2_2017
+verbose   = True and False
+threshold = 'INFO' if verbose else 'WARNING'
+print ">>> run = %s"%run
```

Comment:
doesn't work in python3

Reply:
Should it work in python3 for the PR into CMSSW?

Reply:
@IzaakWN, yes - please fix as requested by David. The migration to Python3 should complete in the next months, because the maintenance of Python2 ended on Jan 1, 2020.

Reply:
Okay, no problem. I tested commit https://github.com/cms-sw/cmssw/pull/29829/commits/b05e06abdd7a999cb4e2fa79ec4ab7635bb1b5a1 with `python3` (instead of `cmsRun`) in the command line, and used `from __future__ import print_function` to make it compatible with python 2.